core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,16 +18,40 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
```

Comment:
Yeah, i think switching it brand icon might be good idea globally. Also in library page the search functionality get hidden in mobile. But if put this css in oppia.css, then search l that is the generally feature that looks weird